nilearn/reporting/utils.py

```diff
@@ -38,6 +38,11 @@ def figure_to_svg_base64(fig):
     return base64.b64encode(figure_to_svg_bytes(fig)).decode()
 
 
+def figure_to_png_base64(fig):
+    """Save figure as png and return it as 64 bytes."""
+    return base64.b64encode(figure_to_png_bytes(fig)).decode()
```

Comment:
This function was removed in #5038. Seems like we never used it before, but we need it to create png images in the reports.